src/wattsi.pas

```diff
@@ -637,7 +636,6 @@    TCrossReferences = record
                         begin
                            SecondLI := NewLI.CloneNode(True);
                            ExtractedData := EnsureID(NewLI, 'toc-' + LastSeenHeadingID);
-                           Assert(not ExtractedData.IsEmpty);
```

Comment:
The point of the assert here is to make sure EnsureID doesn't break without you noticing. :-)